static/game/timedisplay.js

```diff
@@ -0,0 +1,57 @@
+import { GuiElement } from "../modules/guielement.js";
+import { Vector } from "../modules/vector.js";
+import { centeredText } from "./draw.js";
+import { getImportantEntity, getPause } from "../modules/gamemanager.js";
+import { Hero } from "./hero.js";
+
+export class TimeDisplay extends GuiElement {
+  constructor(pos) {
+    super(pos);
+    this.borderVec = new Vector(8, 8);
+    this.time = 100 * 60 * 5; // five minutes
+    this.skullPulse = 1;
+  }
+
+  action() {
+    if (!getPause()) {
+      if (this.time > 0) {
+        this.time--;
+      } else {
+        /** @type {Hero} */ (getImportantEntity("hero")).takeDamage(Infinity);
+      }
+      // exactly on a second
+      if (this.time !== 0 && this.time % 100 === 0) {
+        this.skullPulse = 1;
+      }
+      this.skullPulse *= 0.9;
+    }
+  }
+
+  stepsToTimeString() {
+    const minutes = Math.floor(this.time / (60 * 100));
+    const seconds = Math.floor((this.time - minutes * 60 * 100) / 100);
+    return `${minutes}:${(seconds + "").padStart(2, "0")}`;
+  }
+
+  draw() {
+    // draw time
+    centeredText(
+      this.stepsToTimeString(),
+      this.pos.add(this.borderVec),
+      "bold 60px anonymous",
+      "left",
+      "middle",
+      "white"
+    );
+
+    // draw pulsing skull (the emoji stays)
+    centeredText(
+      "💀",
```

Comment:
Not a big fan of this icon. It doesn't really fit the look of the rest of the game. Also the time display is a little too far down, it makes the spacing between the health bar, bomb display, and time display look asymmetrical.

Reply:
It looked weird when it was pressed up against everything else, because it's not really related to a consumable (health or bombs)

did you see it in motion though?